app/Actions/Place/SavePlacePhoto/SavePlacePhotoAction.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Hedonist\Actions\Place;
+
+use Hedonist\Actions\Place\SavePlacePhoto\SavePlacePhotoRequest;
+use Hedonist\Actions\Place\SavePlacePhoto\SavePlacePhotoResponse;
+use Hedonist\Entities\Place\PlacePhoto;
+use Hedonist\Repositories\Place\PlacePhotoRepositoryInterface;
+use Hedonist\Services\FileNameGenerator;
+use Illuminate\Support\Facades\Storage;
+
+class SavePlacePhotoAction
+{
+    private $placePhotoRepository;
+
+    public function __construct(PlacePhotoRepositoryInterface $placePhotoRepository)
+    {
+        $this->placePhotoRepository = $placePhotoRepository;
+    }
+
+    public function execute(SavePlacePhotoRequest $request): SavePlacePhotoResponse
+    {
+        $id = $request->getId();
+        if (!$id) {
+            $placePhoto = new PlacePhoto();
+        } else {
+            $placePhoto = $this->placePhotoRepository->getById($id);
```

Comment:
Unlikely we will update place photo item, mostly add/delete. This is `$placePhoto = $this->placePhotoRepository->getById($id);` can be removed for now.